xbmc/interfaces/json-rpc/JSONServiceDescription.cpp

```diff
@@ -768,7 +768,7 @@ JSONRPC_STATUS JSONSchemaTypeDefinition::Check(const CVariant &value, CVariant &
       // allowed there is no need to check every element
       if (value.size() < items.size() || (value.size() != items.size() && additionalItems.size() == 0))
       {
-        CLog::Log(LOGDEBUG, "JSONRPC: One of the array elements does not match in type {0}", name.c_str());
```

Comment:
However for both methods, valgrind complained about memory access on already freed bytes and adding c_str fixes it.
Do you think this is a false positive?